pkg/adapter/containers.go

```diff
@@ -304,3 +304,40 @@ func ReadExitFile(runtimeTmp, ctrID string) (int, error) {
 
 	return exitCode, nil
 }
+
+// Attach ...
+func (r *LocalRuntime) Attach(ctx context.Context, c *cliconfig.AttachValues) error {
+	var (
+		ctr *libpod.Container
+		err error
+	)
+
+	if c.Latest {
+		ctr, err = r.Runtime.GetLatestContainer()
+	} else {
+		ctr, err = r.Runtime.LookupContainer(c.InputArgs[0])
+	}
+
+	if err != nil {
+		return errors.Wrapf(err, "unable to exec into %s", c.InputArgs[0])
```

Comment:
The error message does not seem to work for the `--latest` case. Other commands return `Error: unable to get latest container: no such container`.

Reply:
fixed